CMakeLists.txt

```diff
@@ -57,10 +57,11 @@ if( ${TARGET_OS} STREQUAL "Linux" )
 elseif(${TARGET_OS} STREQUAL "Windows" )
   set(OS "WINDOWS")
   set(OSNAME "Windows")
+  add_definitions(-DWIN32 -DHOST_WIN -DWINNT -DTARGET_WIN -DTARGET_WIN_X86)
   if( CMAKE_SIZEOF_VOID_P EQUAL 8 )
-    add_definitions(-DWIN32 -DHOST_WIN -DWIN64 -DWINNT -DTARGET_WIN_X8632)
+    add_definitions(-DWIN64 -DTARGET_WIN_X8664)
   else( CMAKE_SIZEOF_VOID_P EQUAL 8 )
-    add_definitions(-DWIN32 -DHOST_WIN -DWIN64 -DWINNT -DTARGET_WIN_X8632)
+    add_definitions(-DTARGET_WIN_X8632)
```

Comment:
@xoviat, can you have a look above?

Reply:
It seems to be okay.

Reply:
Actually I think `TARGET_WIN` is the problem.